bbn-test-5/babylon-node/README.md

```diff
@@ -0,0 +1,269 @@
+# Babylon Node Setup
+
+## Table of Contents
+
+1. [Install Babylon Binary](#install-babylon-binary)
+2. [Setup Node Home Directory and Configure](#setup-your-node-home-directory-and-configuration)
+4. [Create Keys](#create-keys)
+5. [Sync Node](#sync-node)
+6. [Get Funds](#get-funds)
+
+
+## Install Babylon Binary 
+
+1. Install [Golang 1.21](https://go.dev/dl)
+2. Verify installation:
+
+```shell
+go version
+```
+
+3. Clone and build Babylon:
+```shell
+git clone git@github.com:babylonlabs-io/babylon.git
+cd babylon
+git checkout bbn-test-5  
+make install
+```
+<!-- TODO: testnet tag to be defined -->
+This command does the following:
+- Builds the daemon
+- Compiles all the Go packages in the project
+- Installs the binary 
+- Makes the `babylond` command globally accessible from your terminal
+
+You should expect to see the following output:
+
+```shell
+# Build output will show:
+go install -mod=readonly -tags "netgo ledger mainnet" \
+    -ldflags '\
+        -X github.com/cosmos/cosmos-sdk/version.Name=babylon \
+        -X github.com/cosmos/cosmos-sdk/version.AppName=babylond \
+        -X github.com/cosmos/cosmos-sdk/version.Version=v0.13.0 \
+        -X github.com/cosmos/cosmos-sdk/version.
+        Commit=976e94b926dcf287cb487e8f35dbf400c7d930cc \
+        -X "github.com/cosmos/cosmos-sdk/version.BuildTags=netgo,ledger" \
+        -w -s' \
+    -trimpath ./...
+```
+
+Now that the binary has been successfully installed, 
+let's check the available actions through the `babylond` command:
+
+```shell
+babylond
+Available Commands:
+  add-genesis-account Add a genesis account to genesis.json
+  collect-gentxs      Collect genesis txs and output a genesis.json file
+  comet               CometBFT subcommands
+  config              Utilities for managing application configuration
+  ...
+```
+
+If the `babylond` command isn't found, ensure `$GOPATH/bin` is in your shell's 
+`$PATH`. Add it with:
+
+ ```shell 
+ echo 'export PATH=$HOME/go/bin:$PATH' >> ~/.profile 
+ ```
+
+## Setup your node, home directory, and configuration
+
+Initialize your node and create the necessary configuration directory. 
+This command will generate several important configuration files 
+including `app.toml`, `client.toml`, and `genesis.json`:
+
+```shell
+babylond init <moniker> --chain-id bbn-test-5 --home <path>
+```
+
+The `<moniker>` is a unique identifier for your node (e.g. `node0`).
+The `--home` flag specifies the directory where your node files will be stored (e.g. `--home ./nodeDir`).
+
+After initialization, you'll need to modify two important configuration files:
+
+1. First, open `app.toml` and update these essential settings from what is auto-generated:
+
+```shell
+# Base configuration
+minimum-gas-prices = "0.005ubbn"
```

Comment:
Here, we should explain that these settings will disable IAVL cache to make the node utilize less memory.
In case of a node that will serve heavy RPC query load, these settings shouldn't be used.